src/exception.cr

```diff
@@ -67,6 +73,56 @@ class Exception
 
     io.flush
   end
+
+  protected def os_error=(@os_error)
```

Comment:
`:nodoc:` ?  (Not sure we nodoc protected methods, but I saw some elsewhere that had it)

Reply:
Done, and moved to `SystemError` module